src/screens/Profile/ApprovedPosts.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
```

Comment:
This file is the same as the PendingPosts.js. Can we make one component instead? In TabLayout.js instead of 
```
Approved: ApprovedPosts,
Pending: PendingPosts,
```

we can do somethign like
```
Approved: () => <MyPosts approved={true} />
Pending: () => <MyPosts approved={false} />
```